packages/libs/src/services/solana/transactionHandler.ts

```diff
@@ -20,7 +20,7 @@ type HandleTransactionParams = {
   recentBlockhash?: Nullable<string>
   logger?: Logger
   skipPreflight?: Nullable<boolean>
-  feePayerOverride?: Nullable<PublicKey>
+  feePayerOverride?: Nullable<PublicKey | string>
```

Comment:
why was the `| string` required with this change? not opposed but do we need it?

Reply:
See other comment. Although I suppose I could change the caller to make a public key.. that might be the better play here actually

Reply:
done!